main.tf

```
@@ -20,6 +21,13 @@ resource "datadog_monitor" "default" {
   type                = each.value.type
   tags                = local.tags
 
+  dynamic "monitor_threshold_windows" {
+    for_each = each.value.monitor_threshold_windows != null ? { create : true } : {}
+    content {
```

```suggestion
    for_each = each.value.monitor_threshold_windows != null ? { create : true } : {}
    
    content {
```
Could you add a newline here please?